packages/fx-core/src/plugins/resource/frontend/dotnet/constants.ts

```diff
@@ -76,30 +76,9 @@ export class DependentPluginInfo {
   static readonly solutionPluginName = "solution";
   static readonly subscriptionId = "subscriptionId";
   static readonly resourceGroupName = "resourceGroupName";
-  static readonly resourceNameSuffix = "resourceNameSuffix";
-  static readonly location = "location";
-  static readonly remoteTeamsAppId = "remoteTeamsAppId";
-
-  static readonly functionPluginName = "fx-resource-function";
-  static readonly functionEndpoint = "functionEndpoint";
-
-  static readonly runtimePluginName = "fx-resource-simple-auth";
```

Comment:
@eriolchan @hund030 FYI: @tecton will help drive the changes for VS, which removes the Simple Auth nuget package and use OBO flow to acquire MS Graph token from server side, since the blazor app now runs on server side.
While changing the .NET sdk and scaffolding template, Ning Tang will also help consider how our SDK should change when we support client side blazor app (via Web Assembly), to ensure a smooth change in the future.